lib/upload.js

```diff
@@ -923,15 +924,24 @@ function shouldRetry(err, retryAttempt, options) {
   // - retryDelays option is set
   // - we didn't exceed the maxium number of retries, yet, and
   // - this error was caused by a request or it's response and
-  // - the error is server error (i.e. no a status 4xx or a 409 or 423) and
+  // - the error is server error (i.e. no a status 4xx or a 409 or 423) or
```

Comment:
Can you please help improve this sentence? I think it should be "not a 4XX status except a 409 or 423"